core/kotlinx-coroutines-core/src/main/kotlin/kotlinx/coroutines/experimental/channels/ConflatedBroadcastChannel.kt

```diff
@@ -38,7 +38,7 @@ import kotlinx.coroutines.experimental.selects.SelectInstance
  * [opening][openSubscription] and [closing][SubscriptionReceiveChannel.close] subscription takes O(N) time, where N is the
  * number of subscribers.
  */
-public class ConflatedBroadcastChannel<E>() : BroadcastChannel<E> {
+public class ConflatedBroadcastChannel<E>() : BroadcastChannel<E>, SubscribableVariable<E> {
```

Comment:
Should we inject an automatic item replay into all `ConflatedBroadcastChannel`?
This is a breaking change and a different behavior from channels' default.

In your design `SubscribableVariable` is not a `Channel`, so I prefer to left `ConflatedBroadcastChannel` as is.

Reply:
Currently `ConflatedBroadcastChannel` does re-send the last item through the channel opened with `openSubscription()`. Or am I misunderstanding your point ?

Reply:
Sorry, my fault.

Please discard this comment.